scale-ui/app/modules/jobs/controllers/jobTypesErrorRatesController.js

```diff
@@ -64,6 +88,20 @@
                 }
             });
         };
+        
+        vm.setStyle = function (colField, errorType) {
+            var errorValue = (colField[errorType] / colField.total).toFixed(2);
+            var textColor = errorValue >= 0.5 ? '#fff' : '#000';
+            var rgb = errorType === 'system' ? '103, 0, 13' : errorType === 'algorithm' ? '203, 24, 29' : '241, 105, 19';
```

Comment:
Ideally yes, but the alpha value ("a") in rgba needs to be dynamic and you can't set that individually apart from the rgb values.  Setting the opacity on the element would affect the background and the text, which is not desirable.  Therefore it needs to be done like this.